src/Compilers/CSharp/Portable/Emitter/NoPia/EmbeddedTypesManager.cs

```diff
@@ -572,15 +572,5 @@ protected override CSharpAttributeData CreateCompilerGeneratedAttribute()
             var compilation = ModuleBeingBuilt.Compilation;
             return compilation.TrySynthesizeAttribute(WellKnownMember.System_Runtime_CompilerServices_CompilerGeneratedAttribute__ctor);
         }
-
-        protected override CSharpAttributeData CreateComVisibleAttribute()
-        {
-            var compilation = ModuleBeingBuilt.Compilation;
-            return compilation.TrySynthesizeAttribute(WellKnownMember.System_Runtime_InteropServices_ComVisibleAttribute__ctor,
-                                                      ImmutableArray.Create(new TypedConstant(compilation.GetSpecialType(SpecialType.System_Boolean),
-                                                                                              TypedConstantKind.Primitive,
-                                                                                              value:true)), 
-                                                      isOptionalUse:true);
```

Comment:
Are there any remaining callers that pass `isOptionalUse: true` in C# or VB, or can that parameter be removed in each case?